src/.editorconfig

```diff
@@ -0,0 +1,271 @@
+################################################################
+# Global preferences                                           #
+################################################################
+root = true
+
+# All files
+[*]
+end_of_line = lf
+insert_final_newline = true
+charset = utf-8
```

Comment:
Hah, yeah. I already went down the `utf-8-bom` rabbit hole awhile ago! It's why all the `dotnet format whitespace` commands are commented out for now because, as you say, it spews a lot of warnings. 

Let's discuss as a team at some point, but I think I'd like us to land on just setting `utf-8` everywhere, or at least within `/src/`. There's no reason any of us should need to use Microsoft Word or Notepad for `cs` files :D